designer/server/src/test/resources/local-testing-scala213.override.yml

```diff
@@ -0,0 +1,78 @@
+x-flink-service-common: &common-flink
+  depends_on:
+    flink-scala213-downloader:
+      condition: service_completed_successfully
+  volumes:
+    - /tmp/flink-scala-2-13_2.13-1.1.1-assembly.jar:/opt/flink/lib/flink-scala-2-13_2.13-1.1.1-assembly.jar
+  entrypoint: [
+    "sh", "-c",
+    "rm -f /opt/flink/lib/flink-scala_2.12* && chown flink /opt/flink/lib/flink-scala-2-13_2.13-1.1.1-assembly.jar && /ex-docker-entrypoint.sh \"$@\"", "--"
+  ]
+
+services:
+
+  bootstrap-setup:
```

Comment:
Maybe we could extract `local-testing-common`. Withoug this it is problematic to see what's the difference between scala 2.12 and scala 2.13